src/card.tsx

```diff
@@ -237,49 +237,35 @@ const useVideoWithControls = (
     const videoController = React.useRef<HTMLVideoElement>(null);
     const [videoStatus, dispatch] = React.useReducer(videoReducer, 'loading');
 
-    const onVideoError = () => dispatch('fail');
-    const onVideoPause = () => dispatch('pause');
-    const onVideoPlay = () => dispatch('play');
+    const onVideoError = React.useCallback(() => dispatch('fail'), []);
```

Comment:
personal question I have: will `dispatch` be always the same, 100% guaranteed?

Reply:
Discussed offline